src/adzerk/boot_cljs.clj

```diff
@@ -20,12 +20,22 @@
    none is provided via project"
   (delay (remove pod/dependency-loaded? `[[org.clojure/clojurescript ~cljs-version]])))
 
+(def ^:private QUALIFIERS
+  "Order map for well-known Clojure version qualifiers."
+  { "alpha"  0
+    "beta"   1
+    "rc"     2
+    ""       3})
```

Comment:
Not really important but maybe to keep things compact removing linebreaks would be an option? 
```clj
(def ^:private QUALIFIERS
  "Order map for well-known Clojure version qualifiers."
  { "alpha"  0, "beta" 1, "rc" 2, "" 3})
```